src/CustomTheme/Colors/colors.tsx

```diff
@@ -0,0 +1,21 @@
+export const blue = {
```

Comment:
you can use "import {blue, grey} from  '@mui/material/colors'" in your custom theme itself.